app/Models/Downtime.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ *
+ * @file
+ * @version 0.1
+ * @copyright 2018 CN-Consult GmbH
+ * @author Pascal Stiemer <pascal.stiemer@cn-consult.eu>
+ */
+
+namespace CachetHQ\Cachet\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Downtime extends Model
+{
+    /**
+     * The fillable properties.
+     *
+     * @var string[]
+     */
+    protected $fillable = [
+        'created_at',
+        'resolved_at',
+    ];
+
+    /**
+     * The validation rules.
+     *
+     * @var string[]
+     */
+    public $rules = [
+        'resolved_at'      => 'optional|null',
```

Comment:
'resolved_at' => 'optional|null'
no comma at the end and remove indentation here: at'      =>